src/payments/services/payment.service.ts

```
@@ -219,13 +223,96 @@ export class PaymentService {
+      if (!userDetails) {
+        this.logger.error(`User not found for userId: ${userId}`);
+        return;
+      }
+
+      const firstName = userDetails.firstName || '';
+      const lastName = userDetails.lastName || '';
+
+      // Generate certificate for each target (using contextId as courseId)
+      const certificatePromises = targets.map(async (target) => {
+        const issuanceDate = new Date().toISOString();
+        const expirationDate = '0000-00-00T00:00:00.000Z'; // Default expiration date as per API
```

ESLint

[error] Replace `'0000-00-00T00:00:00.000Z'` with `"0000-00-00T00:00:00.000Z"`

(prettier/prettier)